lib/src/main/java/org/asamk/signal/manager/helper/ProfileHelper.java

```diff
@@ -194,6 +194,9 @@ private Profile getRecipientProfile(RecipientId recipientId, boolean force) {
         }
 
         profile = decryptProfileIfKeyKnown(recipientId, encryptedProfile);
+        if (profile.getGivenName() == null && profile.getFamilyName() == null) {
```

Comment:
why was this added? Even if the names are null the other parts of the profile should still be stored.